src/Components/WebAssembly/testassets/HostedInAspNet.Client/wwwroot/seleniumworkaround.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+  const logs = [];
+
+  console.defaultLog = console.log.bind(console);
```

Comment:
Is there some reason to expose `console.defaultLog` globally? Seems like it could be a private variable here rather than a new member of `console`. Same for the others below.

Also there's no need to do `.bind(...)` if you're going to pass a `this` value to the `.apply` function below.

So altogether this could perhaps be reduced to:

```
const defaultLog = console.log;
```

Reply:
> Is there some reason to expose console.defaultLog globally? 

Nope. I copied this from a SO response. I can get this updated.